src/journeys_utils.js

```diff
@@ -439,7 +439,8 @@ journeys_utils.animateBannerEntrance = function(banner) {
 			banner.style.top = '0';
 		}
 		else if (journeys_utils.position === 'bottom') {
-			if(!journeys_utils.journeyLinkData.journey_link_data['safeAreaRequired']) {
+			var safeAreaRequired = utils.getPropertyValueByNameFromObj(journeys_utils.journeyLinkData['journey_link_data'], 'safeAreaRequired') || false;
```

Comment:
Can we just do: 
```
if (journey_utils.journeyLinkData && journey_utils.journeyLinkData.journey_link_data && journeys_utils.journeyLinkData.journey_link_data['safeAreaRequired'])
```

If necessary to check the key exists, 

add `Object.keys(journeys_utils.journeyLinkData.journey_link_data).hasOwnProperty('safeAreaRequired')`?

Reply:
So the initial error which was coming in was because I was using `journeys_utils.journeyLinkData.journey_link_data` - whenever I use the build.min.js with this, it throws the undefined object error. 

The initial problem was not with the `safeAreaRequired` key but was with the above. 

I ran the test the build.js, so it worked there and I assumed it should be good with minify too, but this fails. Do you have any thoughts on why this could be failing? I can see, in other places also, it is referenced via `[]` and not `.`:  https://github.com/BranchMetrics/web-branch-deep-linking/blob/master/src/journeys_utils.js#L741

Reply:
Additionally, if I go with `Object.keys(journeys_utils.journeyLinkData['journey_link_data']).hasOwnProperty('')` - this gives me false always, irrespective of the key I use, even with `journey_id`.

Reply:
Gotcha, so if that's the case, does the below also fail through minification?

```
if (journey_utils.journeyLinkData && journey_utils.journeyLinkData['journey_link_data'] && journeys_utils.journeyLinkData['journey_link_data']['safeAreaRequired'])
```